ui/search/nuxeo-search-results-layout.js

```diff
@@ -0,0 +1,150 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { dom } from '@polymer/polymer/lib/legacy/polymer.dom.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { pathFromUrl } from '@polymer/polymer/lib/utils/resolve-url.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '../nuxeo-layout.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
+
+{
+  /**
+   * `nuxeo-search-results-layout`
+   * @group Nuxeo UI
+   * @appliesMixin Nuxeo.I18nBehavior
+   * @element nuxeo-search-results-layout
+   */
+  class SearchResultsLayout extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <nuxeo-layout
+          id="results"
+          href="[[_resultsHref(searchName, hrefBase)]]"
+          model="[[_resultsModel(searchName,nxProvider)]]"
+          error="[[i18n('searchResults.layoutNotFound', searchName)]]"
+          on-element-changed="_formChanged"
+        ></nuxeo-layout>
+      `;
+    }
+
+    static get is() {
+      return 'nuxeo-search-results-layout';
+    }
+
+    static get importMeta() {
+      const importMeta = import.meta;
+      importMeta.url = importMeta.url.replace('/er/ui/', '/elements/');
+      return importMeta;
+      // return import.meta;
+    }
+
+    static get properties() {
+      return {
+        /**
+         * The name of the search layout.
+         */
+        searchName: String,
+        /**
+         * The `nuxeo-page-provider` instance used to perform the search.
+         */
+        nxProvider: HTMLElement,
+        /**
+         * An object propagating key/values served by enclosing slot contents.
+         */
+        model: {
+          type: Object,
+          value() {
+            return {};
+          },
+        },
+        /**
+         * The `nuxeo-results` element bound to this element.
+         */
+        results: {
+          type: Object,
+          notify: true,
+        },
+        /**
+         * The base url for the layout href.
+         */
+        hrefBase: String,
+      };
+    }
+
+    get element() {
+      return this.$.results && this.$.results.element;
+    }
+
+    fetch() {
+      if (this.results) {
+        this.results.fetch();
+      } else {
+        return Promise.resolve();
+      }
+    }
+
+    reset() {
+      if (this.results) {
+        this.results.reset();
+      }
+    }
+
+    _resultsHref(searchName, hrefBase) {
+      const name = searchName.toLowerCase();
+      const base = hrefBase || pathFromUrl((this.__dataHost && this.__dataHost.importPath) || this.importPath);
+      return `${base}${name}/${['nuxeo', name, 'search-results'].join('-')}.html`;
+    }
+
+    _resultsModel() {
+      return { nxProvider: this.nxProvider, name: this.searchName };
+    }
+
+    _formChanged(e) {
```

Comment:
who calls this ? 🤔

Reply:
`_formChanged`? It's passed to `<nuxeo-layout>` so that we can fetch the results if the search happens to change, which would trigger the load of a new results layout.
Maybe it's the name that is weird? Should probably be `_resultsChanged` instead, right? This is what we had in Web UI already.